packages/@okta/vuepress-site/docs/guides/configure-access-policy/configure-token-lifetime-per-client/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Configure a custom token lifetime per client
```

Comment:
Suggest changing this to:

> Configure a custom access token lifetime per client

Or modify otherwise to clarify that this is only for access tokens and not id_tokens